lib/cisco_node_utils/cmd_ref/overlay_global.yaml

```diff
@@ -25,9 +25,6 @@ dup_host_mac_detection:
   get_value: '/^l2rib dup-host-mac-detection (\d+) (\d+)$/'
   set_value: "l2rib dup-host-mac-detection <host_moves> <timeout>"
 
-dup_host_mac_detection_default:
```

Comment:
You indicated in the summary that you changed default values.  Are those changes not in the diffs?